src/Files.App/Views/MainPage.xaml

```diff
@@ -215,6 +215,25 @@
 			SelectedItem="{x:Bind SidebarAdaptiveViewModel.SidebarSelectedItem, Mode=TwoWay}"
 			ViewModel="{x:Bind SidebarAdaptiveViewModel}">
 
+			<sidebar:SidebarView.PaneFooter>
+				<sidebar:SidebarItem
```

Comment:
Can you take the AutomationID from the toolbar button and move it here instead?

Reply:
AddressToolbar one named "SettingsButton" tho.
Is that ok?

Reply:
That's fine

Reply:
Sure!